ext/rbs_extension/parser.c

```diff
@@ -963,25 +963,25 @@ static VALUE parse_simple(parserstate *state) {
     return type;
   }
   case kBOOL:
-    return rbs_base_type(RBS_Types_Bases_Bool, rbs_location_current_token(state));
+    return rbs_bases_bool(rbs_location_current_token(state));
```

Comment:
Let's add a note to rename these methods later to a name that is easier to generate so we can avoid some of the complexity in `template.rb`